### PR TITLE
Asymmetric noise: boost velocity, reduce pressure noise

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
Current target noise is [0.01, 0.01, 0.005] for [Ux, Uy, p]. Pressure is the most important metric. Higher noise on velocity channels provides regularization for the shared representation without directly corrupting the pressure signal. By boosting velocity noise 50% while cutting pressure noise 60%, the pressure gradients are cleaner while velocity noise still regularizes the shared layers.

## Instructions
```python
# From:
noise_scale = torch.tensor([0.01, 0.01, 0.005], device=device)
# To:
noise_scale = torch.tensor([0.015, 0.015, 0.002], device=device)
```

That's the only change. Run with `--wandb_group asymmetric-noise`.

## Baseline
- best_val_loss = 2.2155
- val_in_dist/mae_surf_p = 20.24
- val_ood_cond/mae_surf_p = 19.72
- val_ood_re/mae_surf_p = 30.65
- val_tandem_transfer/mae_surf_p = 42.13

---

## Results

**W&B run:** ltjxnlwr
**Epochs completed:** 62 (30-min timeout)
**Peak memory:** 10.6 GB

### Metrics vs Baseline

| Metric | Baseline | This run | Delta |
|---|---|---|---|
| best_val_loss | 2.2155 | 2.3143 | +0.099 worse |
| val_in_dist/mae_surf_p | 20.24 | 22.16 | +1.92 worse |
| val_ood_cond/mae_surf_p | 19.72 | 22.20 | +2.48 worse |
| val_ood_re/mae_surf_p | 30.65 | 31.75 | +1.10 worse |
| val_tandem_transfer/mae_surf_p | 42.13 | 42.42 | +0.29 worse |

### Full surface MAE (best epoch)

| Split | Ux | Uy | p |
|---|---|---|---|
| val_in_dist | 0.3068 | 0.1839 | 22.16 |
| val_ood_cond | 0.2785 | 0.1948 | 22.20 |
| val_ood_re | 0.2804 | 0.2074 | 31.75 |
| val_tandem_transfer | 0.6474 | 0.3427 | 42.42 |

### Volume MAE (val_in_dist)
Ux: 1.3161, Uy: 0.4761, p: 26.65

### What happened

The hypothesis did not work — every metric is worse than baseline. Surface pressure MAE increased across all four splits, and val_loss increased by ~0.10. Reducing pressure noise from 0.005 to 0.002 made things worse, not better.

The rationale was that cleaner pressure targets would improve pressure accuracy. But the result suggests the original pressure noise (0.005) was beneficial as regularization, not harmful. The asymmetric change disrupted a balance — adding more velocity noise did not compensate for the reduced pressure regularization. The model may need noise on the pressure channel to avoid overfitting in-distribution pressure patterns, explaining the degraded OOD performance especially on val_ood_cond (+2.48).

### Suggested follow-ups

1. **Symmetric noise sweep**: Try uniform reductions, e.g. [0.008, 0.008, 0.004] — to see if less noise overall helps.
2. **Zero pressure noise**: Try [0.01, 0.01, 0.0] as an extreme test to see if pressure noise is doing anything at all.
3. **More velocity noise only**: Try [0.02, 0.02, 0.005] — keep pressure noise intact but double velocity noise to test the regularization effect in isolation.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: DANN early progress — EP0.15, all metrics nominal | W\&B `9q77j1wi`

Monitoring W&B directly. At step=1,594 (EP0.15), the DANN implementation looks exemplary:

| DANN Metric | Current value | Interpretation |
|-------------|--------------|----------------|
| domain_loss | **0.701** | ≈log(2)=0.693 → discriminator maximally confused ✓ |
| domain_entropy | **0.693** | Maximum entropy → GRL working correctly ✓ |
| domain_acc | **0.25** | Below chance → backbone hiding domain signal ✓ |
| grl_lambda | **0.00484** | Correctly ramping from 0 |
| domain_n_train | 2 | Train samples per batch feeding discriminator ✓ |
| domain_n_test | 2 | **Test samples feeding discriminator unlabeled ✓** |

No student update needed — I can see from W&B that the run is healthy. Please continue training and **post an update at EP2–3** (step ~21,766–32,649) with val metrics once the first validation epoch completes.

**EP5 kill gate reminder:** `abupt_axis_mean_rel_l2_pct ≤ 7.5%` at step ~54,415 (expected in ~4.8 hours from now, ~15:15 UTC). The run will auto-kill if this threshold is not met.

The key question DANN is answering: can domain-invariant feature learning reduce the val→test vol_p gap (currently ~7pp: val≈3.6%, test≈10.7%)?
